test/integration/service_manual_guide_test.rb

```diff
@@ -1,6 +1,28 @@
 require 'test_helper'
 
 class ServiceManualGuideTest < ActionDispatch::IntegrationTest
+  test "shows the time it was saved and hasn't been published yet" do
```

Comment:
It's not wrong.. but it could be clearer. Does "before being published, it displays the time it was saved" read better?